src/runtime/runtime.cs

```diff
@@ -1793,7 +1784,7 @@ internal static IntPtr PyType_GenericAlloc(IntPtr type, long n)
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern IntPtr _PyObject_GetDictPtr(IntPtr obj);
 
-        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
+        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl, EntryPoint = "_PyObject_GC_New")]
         internal static extern IntPtr PyObject_GC_New(IntPtr tp);
```

Comment:
Is this one used anywhere?

Reply:
None. 
Its implemetantion is 
```c
#define PyObject_GC_New(type, typeobj) \
                ( (type *) _PyObject_GC_New(typeobj) )
```
Just compare to the `PyObject_CheckBuffer`, it's easy to make it be a PInvoke function by specific the entry point.

Reply:
If it's not used, just remove it.

Reply:
Well, it's a not big deal though, but what's bad for declaring a valid api?

Reply:
If we don't use it, it should not be included. The fewer functions we refer to, the easier it will be to keep compatible with different Python versions from the same DLL.

Reply:
That's a point, although I don't think this API will be deleted in the future.
Removed it.